runtime/protos/runtime.proto

```diff
@@ -197,32 +198,30 @@ message ColocationGroup {
 
 // WeaveletInfo contains information provided to a weavelet from its deployer.
 message WeaveletInfo {
-  // TODO(mwhittaker): Remove the notion of a colocation group?
-  // TODO(mwhittaker): Pick a better name for process.
   string app = 1;                            // app name
   string deployment_id = 2;                  // deployment id
   ColocationGroup group = 3;                 // colocation group
   string group_id = 4;                       // colocation group replica id
-  string process = 5;                        // process name
-  string id = 6;                             // weavelet id
-  repeated ComponentGroup same_process = 7;  // See AppConfig.SameProcess.
-  map<string, string> sections = 8;          // See AppConfig.Sections.
-  bool single_process = 9;                   // is a single process?
-  bool use_localhost = 10;                   // listeners listen on localhost?
-  bool process_picks_ports = 11;             // a weavelet picks its own port?
-  string network_storage_dir = 12;           // colocation group storage dir
+  string id = 5;                             // weavelet id
+  // TODO(spetrovic): Rename to same_group.
+  repeated ComponentGroup same_process = 6;  // See AppConfig.SameProcess.
+  map<string, string> sections = 7;          // See AppConfig.Sections.
+  bool single_process = 8;                   // a single-process deployment?
+  bool use_localhost = 9;                    // listeners listen on localhost?
```

Comment:
Oops, I forgot to delete `use_localhost` and `weavelet_pick_ports` here. I'll fix that in another CL. It's not trivial cause there are still some lingering references in the code.

Reply:
Ack!